### DISCUSSION TITLE
为 UIGF标准 添加多游戏支持

### DISCUSSION THREAD

**Discussion by shoucandanghehe** [UIGF-org/.github#5] [Ideas]
崩坏：星穹铁道已经上线，且抽卡系统和原神区别不大
未来绝区零可能也会有类似的系统
是否应该考虑为 UIGF标准 添加一个 `game_type` 之类的字段，以适用于多个游戏

**Comment by Masterain98** [comment]
目前在等下半期卡池出来后，看官方API字段的变化

**Comment by LaoshuBaby** [comment]
https://github.com/LaoshuBaby/HSR-UIG/issues/1

在看到这个Discussion之前贸然的开了一个坑准备征集意见，既然UIGF已经有动作，在打算怎样把那个Issue转移到这边来方便查找

----

此外UIGF组织下有UIGF和UIAF两个标准，是不是组织用一个更包容性的名字更好一些，毕竟未来 **圣遗物**扫描/**遗器**扫描 也都可能出现统一的**物品**可交换标准吧？~~（虽然目前我就只见到yas一个）~~

**Reply by Lightczx** [reply]
域名都持有很久了改组织名肯定是不合适的，同样标准也出台很久了改标准名也是不合适的

**Reply by Lightczx** [reply]
有关游戏内物品的标准名称暂定为UIIF

**Comment by TremblingMoeNew** [comment]
个人认为`game`、`game_biz`和`region_time_zone`都有加的必要
另外我建议HSR版基于2.0/2.1来，2.2和2.3动了`item_id`和`item_type`这两个原始数据这不好

**Reply by Lightczx** [reply]
铁道的记录理论上标准应该要和UIGF分开，原始数据已经很完善了
